editor/src/clj/editor/game_object_non_editable.clj

```diff
@@ -54,12 +54,6 @@
       (project/load-embedded-resource-node project embedded-resource-node-id embedded-resource embedded-resource-pb-map)
       (gu/connect-existing-outputs embedded-resource-node-type embedded-resource-node-id host-node-id embedded-component-connections))))
 
-(g/defnk produce-embedded-component-build-targets [embedded-component-build-targets]
-  (into []
-        (map (comp #(assoc % :resource (bt/make-content-hash-build-resource %))
-                   util/only-or-throw))
-        embedded-component-build-targets))
-
```

Comment:
We don't need to do this anymore. The `:data` of the `MemoryResources` in the `embedded-component-build-targets` will already reflect their `content-hash`.